.license-tools-config.json

```diff
@@ -0,0 +1,42 @@
+{
+  "author": {
+    "name": "Chair for Design Automation, TUM\nCopyright (c) 2025 Munich Quantum Software Company GmbH",
+    "years": [2023, 2025]
+  },
+  "force_author": true,
+  "license": "MIT",
+  "title": false,
+  "include": ["**/*"],
+  "style_override_for_suffix": {
+    ".pyi": "DOCSTRING_STYLE",
+    ".in": "POUND_STYLE",
+    ".mlir": "SLASH_STYLE",
+    ".td": "SLASH_STYLE"
+  },
+  "exclude": [
+    "^\\.[^/]+",
+    "/\\.[^/]+",
+    ".*\\.qasm",
+    ".*\\.md",
+    ".*\\.bib",
+    ".*\\.cff",
+    ".*\\.css",
+    ".*\\.json",
+    ".*\\.html",
+    ".*\\.tfc",
+    ".*\\.qc",
+    ".*\\.real",
+    ".*\\.tex",
+    ".*\\.toml",
+    ".*\\.yaml",
+    "uv\\.lock",
+    "py\\.typed",
+    ".*build.*",
+    ".*\\.rst",
+    ".*\\.ipynb",
+    "LICENSE",
```

Comment:
If you rename the license file to `LICENSE.md`, you can remove this exclude and the license file will be formatted a little bit nicer.
This would probably also be a good time to adopt PEP 639 license expressions.
That is, in pyproject.toml: https://github.com/munich-quantum-toolkit/core/blob/72e5e2c993b82367966b7b5f6cbafd63cad653a4/pyproject.toml#L13-L14 and removing the Trove classifier for the license right below that.